cartography/intel/aws/emr.py

```diff
@@ -60,9 +61,8 @@ def load_emr_clusters(
         aws_update_tag: int,
 ) -> None:
     logger.info("Loading EMR %d clusters for region '%s' into graph.", len(cluster_data), region)
-
+    ensure_indexes(neo4j_session, EMRClusterSchema())
```

Comment:
Can you try to sneak this into the `NodeSchema`'s `__init__`, so that we don't have to call this? If not, maybe call this within `build_ingestion_query` or `load_graph_data`?